src/main/java/io/datanerds/avropatch/value/conversion/DateConversion.java

```diff
@@ -0,0 +1,66 @@
+package io.datanerds.avropatch.value.conversion;
+
+import io.datanerds.avropatch.schema.CustomTypes.DateType;
+import org.apache.avro.Conversion;
+import org.apache.avro.LogicalType;
+import org.apache.avro.LogicalTypes;
+import org.apache.avro.Schema;
+import org.apache.avro.generic.GenericFixed;
+
+import java.nio.ByteBuffer;
+import java.util.Date;
```

Comment:
Hm .. just a general question .. could we built a DateConversion for the yodatime DateTime?